code/game/gamemodes/cult/runes.dm

```diff
@@ -30,6 +30,12 @@ var/list/sacrificed = list()
 		if (istype(user, /mob/living))
 			user.take_overall_damage(5, 0)
 		qdel(src)
+
+	var/turf/T = get_turf(user)
+	if (T.z in current_map.admin_levels)

```

Comment:
I would move that to the general rune activation to not only limit the teleport rune, but all other runes aswell.

Reply:
My original plan was just to stop people from teleporting, you can still have someone write a rune in an admin z-level, like a merchant cultists sending a message or something.

Reply:
Hm, I still feel that the ability to use runes on the merchant station should be completely removed.
(For the reasons explained in the topic)
If not, then the summon narnar rune should definitly be restricted.

Reply:
Alright, I will make so that you can only summon nar'sie on the station levels.

Reply:
Done, you now need to be on one of the station z-levels to summon nar'sie.